api/soap/mc_api.php

```diff
@@ -797,12 +797,14 @@ function mci_get_category( $p_category_id ) {
 }
 
 /**
- * Convert a category name to a category id for a given project
+ * Convert a category name, or category object reference (array w/ id, name,
+ * or id + name) to a category id for a given project.
+ *
```

Comment:
Suggest to clarify in the PHPDoc block(s) the difference between *mci_get_category_id*  and the "internal" variant, when to use one or the other.

Reply:
Maybe I should change the `_internal` variant to an anonymous method with the regular one since no one else is expected to call the internal function.

Reply:
That would work too, I guess. Your call.